packages/edit-navigation/src/components/header/style.scss

```diff
@@ -1,25 +1,61 @@
 .edit-navigation-header {
+	padding: $grid-unit-20;
+	background: #f1f1f1; // Need to find a variable for this
+
+	h1 {
+		font-size: 23px;
+		font-weight: 400;
+		line-height: 1.3;
```

Comment:
Are there any existing variables for these?

Reply:
I'd think so; I didn't look. I copied this from the Pages section.

Reply:
`$default-line-height` is 1.4 which is close enough.

bf6aefae50